### PR TITLE
[DX-965] Operator release notes

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

Add Operator 0.16 release notes

---

### Preview Link

<br>

https://deploy-preview-3906--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-operator/release-notes/operator-0.16/

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+description: "Release notes documenting updates, enhancements, fixes and changes for Tyk Operator versions within the 0.16.x series."
+---
+**Open Source ([Mozilla Public License](https://github.com/TykTechnologies/tyk/blob/master/LICENSE.md))**
+
+**This page contains all release notes for version 0.16 displayed in reverse chronological order**
+
+## Support Lifetime
+Our minor releases are supported until our next minor comes out. 
+
+## 0.16.0 Release Notes
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md:13]:
```suggestion
## 0.16.0 Release Notes bla
```

**Comment by letzya** [comment]
@caroltyk We missed the release date. Let's try to get it next time ready and merged before the release :)
Also - best to assign Simon and create a an RN jira

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+##### Release date 12 Jan 2024
+
+#### Breaking Changes
+This release has no breaking changes.
+
+#### Deprecations
+There are no deprecations in this release.
+
+#### Upgrade instructions
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md]:
```suggestion
#### Upgrade Instructions
```

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+#### Changelog {#Changelog-v0.16.0}
+
+##### Added
+
+<ul>
+<li>
+<details>
+<summary>Added imagePullSecrets configuration for ServiceAccount in Tyk Operator Helm chart </summary>
+
+  Added imagePullSecrets configuration for ServiceAccount in Tyk Operator Helm chart
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md]: Maybe update with a link for [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) to external K8s site and add a sentence explaining purpose of what this change allows the user to do.

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+</details>
+</li>
+</ul>
+
+<ul>
+<li>
+<details>
+<summary>Added support of UDG Global Header in ApiDefinition CRD </summary>
+
+Added to ApiDefinition CRD: support for UDG Global Header at [spec.graphql.engine.global_headers](https://doc.crds.dev/github.com/TykTechnologies/tyk-operator/tyk.tyk.io/ApiDefinition/v1alpha1@v0.16.0#spec-graphql-engine-global_headers) object in ApiDefinition CRD
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md]: Is the _UDG Global Header_ feature only compatible with Tyk 5.2 and, if so should it be mentioned here?

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+</ul>
+
+##### Fixed
+
+<ul>
+<li>
+<details>
+<summary>Fixed CVE-2023-39325 (NVD) </summary>
+
+Fixed [CVE-2023-39325 (NVD)](https://nvd.nist.gov/vuln/detail/CVE-2023-39325)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md:111]: Is this a dependency tracked CVE or an internal scanned CVE?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @caroltyk PR looks good to me, made minor suggestions and then will merge and release. 

Double checking if the CVE on the release note is internally scanned CVE? If so then policy is to not include on release notes, only external CVEs should be included.

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md (hunk 3 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md]:
```suggestion
  Added [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) configuration for ServiceAccount in Tyk Operator Helm chart. It allows user to pull image from a private registry.
```

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md (hunk 4 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md]:
```suggestion
Added to ApiDefinition CRD: support for UDG Global Header at [spec.graphql.engine.global_headers](https://doc.crds.dev/github.com/TykTechnologies/tyk-operator/tyk.tyk.io/ApiDefinition/v1alpha1@v0.16.0#spec-graphql-engine-global_headers) object in ApiDefinition CRD. This feature is compatible with Tyk 5.2 or above.
```

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md (hunk 5 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.16.md:111]: It was reported by dependabot and dependency tracked

**Review by dcs3spp — APPROVED** [review]
@caroltyk PR looks good to me, I'll merge and release

**Pull request merged**